model_analyzer/config/run/run_config.py

```diff
@@ -17,7 +17,7 @@
 
 class RunConfig:
     """
-    Encapsulates all the information needed to run one or more models 
```

Comment:
I was ignoring white space before. This needs to be reverted as well (there should be no changes to this file in your PR)